workspaces/topology/plugins/topology/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ### Dependencies
 
+## 2.0.0
+
+### Major Changes
+
+- 730359c: Migrated to MUI5
```

Comment:
I don't expect that we want increase the version to 2.0 for that UI library update. @ciiay  Can we decrease the changeset to a minor change? Wdyt?

Reply:
Raised a [PR](https://github.com/backstage/community-plugins/pull/1774) to address this, thanks.